drivers/hmis/app/graphql/types/hmis_schema/referral_posting.rb

```diff
@@ -22,6 +22,7 @@ class HmisSchema::ReferralPosting < Types::BaseObject
     # Fields that come from ReferralHouseholdMembers
     field :hoh_name, String, null: false
     field :hoh_mci_id, ID, null: true
+    field :hoh_client, HmisSchema::Client, null: false
```

Comment:
This needs to be nullable. It wont be present if the referral hasn't been accepted